notebooks/client demo.ipynb

```diff
@@ -1,14 +1,154 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lfview.resources import files, manifests, spatial, scene"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from seequent.resources import files, manifests, spatial, scene"
+    "from lfview import client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session = client.Session(api_key='eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOiIxNTYxNjgyNDUwIiwic2NvcGVzIjpbIm9yZ3MucHJvamVjdHMucmVhZCIsInByb2plY3Qudmlld3Mud3JpdGUiLCJwcm9qZWN0Lm1lbWJlcnMucmVhZCIsInByb2plY3Quc3BhdGlhbC5yZWFkIiwidmlldy5tZW1iZXJzLnJlYWQiLCJ1c2VyLmluZm8ucmVhZCIsInByb2plY3Quc3BhdGlhbC53cml0ZSIsIm9yZ3MubWVtYmVycy5yZWFkIiwidmlldy5tZW1iZXJzLndyaXRlIiwicHJvamVjdC5maWxlcy53cml0ZSIsInByb2plY3QuZmlsZXMucmVhZCIsInZpZXcubWVtYmVycy5kZWxldGUiLCJ1c2VyLnRlcm1zLmFjY2VwdCIsIm9yZ3MucHJvamVjdHMud3JpdGUiLCJ1c2VyLm9iamVjdHMucmVhZCIsInByb2plY3Qudmlld3MuZGVsZXRlIiwicHJvamVjdC52aWV3cy5yZWFkIiwicHJvamVjdC52aWV3cy5jcmVhdGUiXSwic2lkIjoiZXBkZ20xdWlrYnV5cmJ6eW8zZGoiLCJ1aWQiOiJhZGFlMXhuZW1pbHk3cWc4cmh4eCIsImVtYWlsIjoibWF0dGhhdXMud29vbGFyZCtvY204MGd3bnJnZXBydDVieHFjbTJ3Y3A1bmlyZXFvNEBzZWVxdWVudC5jb20iLCJkaXNwbGF5X25hbWUiOiJUXHUwMTAzbmcgUGVsY292XHUwMGUxIiwibW9kZSI6InByaXZpbGVnZWQiLCJleHAiOiIxNTYxNzQyNDUwIiwiaXNzIjoibWUifQ.FcnCHO20ZWnsQGqO3ehwWtRX5dRktXiq-W4m5_2SIs4', endpoint='http://localhost:8910')"
```

Comment:
Can you remove the api key here and clear output on the notebook?

Reply:
sorry did not notice I had commute changes to the notebook :/